docs/getting_started.md

```diff
@@ -403,4 +403,4 @@ Congrats! You made it to the end of the annotated MNIST example. You can revisit
 the same example, but structured differently as a couple of Python modules, test
 modules, config files, another Colab, and documentation in Flax's Git repo:
 
-https://github.com/google/flax/tree/main/examples/mnist
+[https://github.com/google/flax/tree/main/examples/mnist](https://flax.readthedocs.io/en/latest/getting_started.html)
```

Comment:
could we link to the github page for mnist directly here? (ie the same url)

Reply:
Perfect I think I have added that.